src/main/java/com/miraisolutions/xlconnect/Workbook.java

```diff
@@ -1483,19 +1495,19 @@ public int getLastColumn(String sheetName) {
         return getLastColumn(getSheet(sheetName));
     }
 
-    public void appendNamedRegion(DataFrame data, String name, boolean header) {
+    public void appendNamedRegion(DataFrame data, String name, boolean header, boolean overwriteFormulaCells) {
         Sheet sheet = workbook.getSheet(getName(name).getSheetName());
         // top, left, bottom, right
         int[] coord = getReferenceCoordinates(name);
-        writeData(data, sheet, coord[2] + 1, coord[1], header);
+        writeData(data, sheet, coord[2] + 1, coord[1], header, overwriteFormulaCells); // TODO needed ?
```

Comment:
```suggestion
        writeData(data, sheet, coord[2] + 1, coord[1], header, overwriteFormulaCells);
```
As seen by the updates in https://github.com/miraisolutions/xlconnect/pull/175, this is indeed required for appendNamedRegion. But we could consider removing it for appendWorksheet where the argument doesn't make any sense (or we control it in XLConnect by not exposing it to the R interface, not sure what is the preferred way).

Reply:
:+1: